3_visualize/src/plot_inset.R

```diff
@@ -7,59 +7,81 @@ plot_inset <- function(station_data, station, us_data, regions, region_sf, focal
       st_cast("MULTIPOLYGON") %>%
       st_cast("POLYGON")
   }
-    
+  
```

Comment:
Great changes here. The mobile version looks great.

![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/5223dbb2-b606-4adc-9940-c6201ef484f0)

I suggest we drop the gage locations from the *regional* map on desktop, too, so that it looks more like the mobile one above. Right now, the gages get loaded only on hover and it's distracting. So, better to just drop them here so it's less distracting and has more visual continuity with the mobile version.

Reply:
Good thought. They are now dropped

Reply:
![image](https://github.com/DOI-USGS/drought-timeline/assets/54007288/fdefc947-978b-4063-b180-4e1a3e00bedc)